### PR TITLE
release-check-list: Update release checklist

### PR DESCRIPTION
**Pull request by stevenhorsman**
- Merge steps that can be achieved in a common PR
- Add steps to wait for runtime payload builds
- Add peer pods steps

### CODE REVIEW

**Review by fitzthum — APPROVED** [review]
Nice. I will probably make a more aggressive PR later, but these are all good changes imo.

File: .github/ISSUE_TEMPLATE/release-check-list.md
Comment (wainersm) [inline .github/ISSUE_TEMPLATE/release-check-list.md:46]: s/containers/contains ? Yeah, it is containers everywhere all the time :)

File: .github/ISSUE_TEMPLATE/release-check-list.md
Comment (wainersm) [inline .github/ISSUE_TEMPLATE/release-check-list.md:64]: Just for me to understand the recent changes: the runtime-payload is now multi-arch. The s390x overlay is only needed currently to restrict the runtimeClasses (i.e. makes no sense to have, say, kata-qemu-sev on s390x).

File: .github/ISSUE_TEMPLATE/release-check-list.md
Comment (wainersm) [inline .github/ISSUE_TEMPLATE/release-check-list.md:122]: extract space pods\s\srelease

Comment (wainersm) [reply .github/ISSUE_TEMPLATE/release-check-list.md:122]: BTW, I just created an issue (https://github.com/confidential-containers/cloud-api-adaptor/issues/1108) suggesting that we have a release checklist on peer pods as well. We could have the content of this bullet detailed there, and then just link here.

File: .github/ISSUE_TEMPLATE/release-check-list.md
Comment (wainersm) [inline .github/ISSUE_TEMPLATE/release-check-list.md:145]: I spoken with @jensfr  and I will take over this task.

File: .github/ISSUE_TEMPLATE/release-check-list.md
Comment (stevenhorsman) [reply .github/ISSUE_TEMPLATE/release-check-list.md:46]: Thanks for spotting this - at least I didn't autocomplete to containerd that I have a habit of doing 😿

File: .github/ISSUE_TEMPLATE/release-check-list.md
Comment (stevenhorsman) [reply .github/ISSUE_TEMPLATE/release-check-list.md:64]: Yeah - that was a change I introduced a few weeks ago in https://github.com/confidential-containers/operator/pull/169/commits/44ff27b35bf3917145057afc85810e97b7e72d97

**Comment by stevenhorsman** [comment]
@wainersm - thanks for the review and sorry for my poor english. I've kept the updates as a separate commit for ease of re-review, but if/when you are happy let me know and I'll fix-up them.

**Comment by wainersm** [comment]
> @wainersm - thanks for the review and sorry for my poor english. I've kept the updates as a separate commit for ease of re-review, but if/when you are happy let me know and I'll fix-up them.

ywc @stevenhorsman ! My perfect english skills I put in your service ;)

**Review by wainersm — APPROVED** [review]
Thanks @stevenhorsman !

File: .github/ISSUE_TEMPLATE/release-check-list.md
Comment (stevenhorsman) [reply .github/ISSUE_TEMPLATE/release-check-list.md]: I'm going to suggest waiting for a day or so to merge this and I'll try and move/write the release process in the cloud-api-adaptor and then link to that as suggested.

Comment (wainersm) [reply .github/ISSUE_TEMPLATE/release-check-list.md]: Works for me.

Comment (stevenhorsman) [reply .github/ISSUE_TEMPLATE/release-check-list.md]: I've created https://github.com/confidential-containers/cloud-api-adaptor/pull/1110 for this, so once it gets reviewed and merged, we can link to sections in there.

**Pull request merged**
